src/main/java/org/gridsuite/study/server/service/SupervisionService.java

```diff
@@ -171,7 +176,23 @@ private Integer deleteSensitivityAnalysisResults() {
         Map<UUID, String> subreportToDelete = formatSubreportMap(StudyService.ReportType.SENSITIVITY_ANALYSIS.reportKey, nodes);
         reportService.deleteTreeReports(subreportToDelete);
         sensitivityAnalysisService.deleteSensitivityAnalysisResults();
+
         LOGGER.trace("{} results deletion for all studies : {} seconds", ComputationType.SENSITIVITY_ANALYSIS, TimeUnit.NANOSECONDS.toSeconds(System.nanoTime() - startTime.get()));
+
+        return nodes.size();
+    }
+
+    private Integer deleteNonEvacuatedEnergyAnalysisResults() {
```

Comment:
Shouldn't we also update this file https://github.com/gridsuite/admin-tools/blob/main/delete_computation_results.py 
so that when we want to delete all study results, we also delete non evacuated energy results?

Reply:
Yes. Another PR has been created for this ...